core/utils/pcap_handle.h

```diff
@@ -0,0 +1,39 @@
+#ifndef PCAP_HANDLE_H
```

Comment:
You might change this to:
#ifndef BESS_CORE_UTILS_PCAP_HANDLE_H_

(and the subsequent #define)